### PR TITLE
Full Auto 10/22

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Full auto 10/22s"

#### Purpose of change

Full auto 10/22s are possible to manufacture by hand, and would be a nice addition to the cataclysm. 

#### Describe the solution

Adds recipes and guns for the 10/22 and the Charger. The charger is just a 10/22 in a pistol form. Since you're directly modifying the gun, a drop in autosear wouldn't work. Also learnt by the Anarchist Cookbook, even though it doesn't include it. Just fit thematically, and I don't want to add a whole book on homemade/modified firearms (just yet). 

#### Describe alternatives you've considered

Not doing it.

#### Testing
image
image

#### Additional context

Requires #74370 first 
https://ia601300.us.archive.org/23/items/Ruger_10-22_Full-Auto_Conversion_version_2/Ruger_10-22_Full-Auto_Conversion_version_2_text.pdf
If this is unwanted/dangerous please delete it or tell me to do so

### CODE REVIEW

File: data/json/items/gun/22.json [hunk 1]
```diff
@@ -207,6 +207,23 @@
...
+    "copy-from": "ruger_1022",
+    "type": "GUN",
+    "name": "full-auto varmint rifle",
+    "description": "A handy and lightweight rifle, modified to fire semi and full auto.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "ruger_1022_auto",
+        "name": { "str": "full-auto Ruger 10/22 rifle" },
+        "description": "A Ruger 10/22 modified post-cataclysm to selective fire.  While the .22 round isn't very powerful, full auto makes it a lot more effective, and still easily controllable in full auto."
```
Comment (TheShadowFerret) [inline data/json/items/gun/22.json:221]:
```suggestion
        "description": "A Ruger 10/22 modified post-Cataclysm to selective fire.  While the .22 round isn't very powerful, full auto makes it a lot more effective, and still easily controllable in full auto."
```

File: data/json/items/gun/22.json [hunk 2]
```diff
@@ -552,5 +569,22 @@
...
+    "copy-from": "ruger_charger",
+    "type": "GUN",
+    "name": "full-auto varmint pistol",
+    "description": "A lightweight varmint pistol, modified to fire both semi, and full auto.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "ruger_charger_auto",
+        "name": { "str": "full-auto Ruger 22 Charger" },
+        "description": "A Ruger 22 Charger modified post-cataclysm to selective fire.  While the .22 round isn't very powerful, full auto makes it a lot more effective, and still easily controllable in full auto even with the lack of a stock."
```
Comment (TheShadowFerret) [inline data/json/items/gun/22.json:584]:
```suggestion
        "description": "A Ruger 22 Charger modified post-Cataclysm to selective fire.  While the .22 round isn't very powerful, full auto makes it a lot more effective, and still easily controllable in full auto even with the lack of a stock."
```

**Review by TheShadowFerret — COMMENTED** [review]
>If this is unwanted/dangerous please delete it or tell me to do so

_Home Expedient Firearms_ is already floating around here because of the Luty 😛

**Comment by Holli-Git** [comment]
> _Home Expedient Firearms_ is already floating around here because of the Luty 😛

Oh I know, just wanted to make sure. Also the effort and effectiveness of a Luty is a whole lot different than a full auto 10/22

**Comment by Zireael07** [comment]
Source needed for "full auto can be manufactured by hand"

**Comment by Holli-Git** [comment]
> Source needed for "full auto can be manufactured by hand"

Look in additional context

**Pull request merged**